README.md

```diff
@@ -98,6 +98,26 @@ subscript(index: Int) -> T {
 
 _Rationale:_ The intent and meaning of the first version is clear, and results in less code.
 
+#### Avoid redundant type declarations.
+
+When type information can be inferred from an initial value, avoid declaring it explicitly, unless doing so is an active decision to improve code legibility.
+
+```swift
+var isOnFire = false
+var lookupTable = [String: Int]()
```

Comment:
If we're going to decide that this _is_ how we should be initializing empty collections I'll make a few additional edits. I think it makes sense, though. I've been using:

```swift
var lookupTable = [String: Int]()
```
but I'm on the fence. This (above) is shorter, and it avoids using the slightly-esoteric `[]` and `[:]` notation for initializing empty collections, but it is also essentially adding a type annotation but choosing to do it in the 'wrong' place. 

maybe my argument against myself would be that I prefer `var aCGFloat: CGFloat = 2` to `var aCGFloat = CGFloat(2)`. 


And perhaps the rule that explains this best is something like: "If type information cannot be inferred from the type's constructor (as in when constructing an `IntegerLiteralConvertable` from an `Int` literal, or an empty collection with `[]` or `[:]`) then type information should be added to the declaration, not to the constructor." And then an example. Is this clear to you? This feels like almost a separate rule.